src/buildHTML.js

```diff
@@ -524,6 +526,8 @@ groupTypes.array = function(group, options, prev) {
     var nc = 0;
     var body = new Array(nr);
 
+    var style = options.style;
```

Comment:
This is called `style` here, `fstyle` in `genfrac` and `options.style` in supsub. Perhaps we should use the same name in all three of these functions.